kubernetes/annotations_and_labels.md

```diff
@@ -8,6 +8,9 @@ This page defines common annotations and labels we set in Kubernetes objects.
   location. For now this should be in our `giantswarm/giantswarm` repository in
   which we crosslink all pages to create a reasonable documentation of all kinds
   of Kubernetes resources, their functionality and relationships.
+- `giantswarm.io/restart` - this annotation is meant to be used for DaemonSet,
+  Deployment and StatefulSet restarts. The value can be arbitrary string that
+  works for given reconciliation purposes.
```

Comment:
```suggestion
  works for given reconciliation purposes. It should be set manually by the user. After full restart the annotation is removed by restarting authority.
```

Reply:
No, in general this should not be used manually but programmatically (e.g. by `azure-operator` in my case). It also isn't supposed to be removed afterwards because that would trigger another rolling restart.

Reply:
Alright. So this is just to change the DS/StatefulSet so to rolling update is triggered. With you now.